pkg/restapi/vc/operation/operations.go

```diff
@@ -111,11 +181,61 @@ func createCredential(data *CreateCrendential) (*verifiable.Credential, error) {
 	return validatedCred, nil
 }
 
+func (c *Operation) createProfile(pr *ProfileRequest) (*ProfileResponse, error) {
+	if pr.DID == "" && pr.URI == "" {
+		return nil, fmt.Errorf("missing DID and URI information")
+	}
+
+	u, err := prepareURL(pr.URI)
+	if err != nil {
+		return nil, err
+	}
+
+	issueDate := getIssueDate()
+	profileResponse := &ProfileResponse{uuid.New().String(), u, &issueDate}
+
+	err = c.credentialStore.SaveProfile(profileResponse)
+	if err != nil {
+		return nil, err
+	}
+
+	return profileResponse, nil
+}
+func prepareURL(uri string) (string, error) {
+	u, err := url.Parse(uri)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse the uri: %s", err.Error())
+	}
+
+	u.Scheme = "https"
```

Comment:
Might there be issues with hardcoding the scheme to https?

Reply:
I am not sure about the value of this function. My understanding was that you will add credential id to whatever is stored in profile e.g. https://example.com/credentials will be stored in the profile and you will add /1234 at the end